packages/rashell/rashell.0.1.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "1.2"
+version: "0.1.0"
+maintainer: "michipili@gmail.com"
+authors: "Michael Grünewald"
+license: "LGPL"
```

Comment:
LGPL is equivalent to GPL for OCaml libs.

Described here: https://github.com/vincenthz/ocaml-sha/issues/17 for example

@avsm is there a canonical explanation for this somewhere?